### DISCUSSION TITLE
i'm really excited!

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#856] [(A) Anything]
I'm really excited, the reason for this is because me & my friend made our own version of a interactive to do list and 12 month calendar! (not by myself of course) the people who inspired me to start getting better on HTML are ChickenOnYT &  Irhrhd, so thanks for inspiring me to start learn HTML + CSS!

P.S: [if you want to see the to do list plus the code you can go here!](https://github.com/UN3N0WNGH0ST/to-do-list)

**Comment by ghost** [comment]
didn't UN3N0WNGH0ST make this @SakuraKitsunee ???

**Reply by SakuraKitsunee** [reply]
i am UN3N0WNGH0ST i just changed my username

**Reply by ghost** [reply]
since when could you change a github username?

**Reply by ghost** [reply]
LOL

**Reply by ghost** [reply]
I just found out how

**Comment by NX721** [comment]
Lmao

**Comment by ghost** [comment]
@SakuraKitsunee what is your profile??

**Reply by SakuraKitsunee** [reply]
my pfp its me irl

**Reply by SakuraKitsunee** [reply]
at the gym

**Reply by ghost** [reply]
ohh kk

**Reply by ghost** [reply]
k i couldn't tell it was a little blury

**Reply by ghost** [reply]
it was unclear

**Reply by ghost** [reply]
but I see it now

**Reply by ghost** [reply]
kk thx

**Reply by ghost** [reply]
this isn't related to the thread but why are you standing like that @SakuraKitsunee    o(〃＾▽＾〃)o 

@SakuraKitsunee

**Reply by SakuraKitsunee** [reply]
i was trying to look cute lol

**Reply by SakuraKitsunee** [reply]
@PatheticBurger

**Reply by SakuraKitsunee** [reply]
btw do you have discord?

**Reply by SakuraKitsunee** [reply]
if so add SakuraKitsunee#5271

**Reply by ghost** [reply]
No, i don't have discord sry /ᐠ｡ꞈ｡ᐟ\ 

:(

**Reply by ghost** [reply]
but thx anyway (^・ω・^ )

**Reply by ghost** [reply]
I changed my username ¯\_(ツ)_/¯

**Reply by SakuraKitsunee** [reply]
i like it!

**Reply by ghost** [reply]
@SakuraKitsunee Why is your profile picture you wearing very few clothes

**Reply by SakuraKitsunee** [reply]
@Hamza123-png what do you mean i'm just wearing leggings and a crop-top

**Reply by ghost** [reply]
what is a "crop-top"

**Reply by ghost** [reply]
@SakuraKitsunee  What is a "top crop"

**Reply by SakuraKitsunee** [reply]
@Hamza123-png its kinda like a skirt for your chest

**Reply by SakuraKitsunee** [reply]
![a54eed973b189947814967b3af959638](https://user-images.githubusercontent.com/75956379/115898986-12b33380-a413-11eb-8305-99237743e190.jpg)
most of them look like this

**Reply by ghost** [reply]
but isn't that a scam @SakuraKitsunee because a normal shirt is bigger and makes you warmer

**Reply by SakuraKitsunee** [reply]
no not really, we girls like to show skin and look cute for the boys

**Reply by ghost** [reply]
o ok but it only shows a little bit so what is the point of showing a lot of skin or a little @SakuraKitsunee ?

**Reply by SakuraKitsunee** [reply]
no actually it shows a pretty good amount soooo...

**Reply by ghost** [reply]
idk im not a girl so I don't know if that is alot

**Reply by ghost** [reply]
I wouldn't know

**Reply by ghost** [reply]
@Hamza123-png bruh ur weird

**Reply by ghost** [reply]
@Hamza123-png no offence tho

**Reply by ghost** [reply]
@SakuraKitsunee what is a "good" amount, a little is good or is a lot good

> no actually it shows a pretty good amount soooo...

**Reply by ghost** [reply]
also why do you have to look cute @SakuraKitsunee

**Reply by SakuraKitsunee** [reply]
1. in the middle of a little and a lot!
2.  for the Instagram and for my boyfriend

**Reply by SakuraKitsunee** [reply]
@Hamza123-png

**Reply by ghost** [reply]
what is your instagram?

**Reply by ghost** [reply]
@SakuraKitsunee

**Reply by SakuraKitsunee** [reply]
no offense, but i don't want to give it to someone who I barely know, srry

**Reply by ghost** [reply]
@SakuraKitsunee wait did that guy delete his account?

**Reply by SakuraKitsunee** [reply]
is the ghost account for deleted users, if so then yeah

**Reply by SakuraKitsunee** [reply]
@BurnedWaffle yeah he did

**Comment by SakuraKitsunee** [comment]
@Hamza123-png
